CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## master
 
 - [Fixes [#75](https://github.com/palkan/logidze/issues/70)] Fix association versioning with an optional belongs to ([@ankursethi-uscis][])
+- [PR [#79](https://github.com/palkan/logidze/pull/13)] Allow adding meta information to versions using `with_meta` (addressed [Issue [#60]](https://github.com/palkan/logidze/issues/60)). ([@DmitryTsepelev][])
```

Comment:
Let's add an example here.

And also a note about how to upgrade form the older versions

Reply:
Done, added a note about the change in the data structure, if someone works with the `version.data['r']` directly he'll have to make the fallback manually, but otherwise no actions needed to upgrade

Reply:
> otherwise no actions needed to upgrade

Users have to regenerate logidze stored functions, don't they? Thus they have to create an upgrade migration https://github.com/palkan/logidze#upgrade-from-previous-versions

Reply:
Oh, right, added it too